src/gui/mzroll/eicwidget.cpp

```diff
@@ -1843,9 +1858,11 @@ void EicWidget::contextMenuEvent(QContextMenuEvent * event)
 	QAction* o3 = options.addAction("Show TIC");
 	o3->setCheckable(true);
 	o3->setChecked(_showTicLine);
+    
 	connect(o3, SIGNAL(toggled(bool)), SLOT(showTicLine(bool)));
 	connect(o3, SIGNAL(toggled(bool)), SLOT(replot()));
 
+
```

Comment:
@MdAzmatAli Please try not to add blank lines where there is no code change. Function  _contextMenuButton_  has no change other than these two blank lines. Please try to avoid these.

Reply:
fixed here, and will keep this in mind going forward